lectures/integration/notebook.ipynb

```diff
@@ -919,7 +919,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def problem_genz_discontinuous_no_defaults(x, u, a):\n",
+    "def genz_discontinuous_no_defaults(x, u, a):\n",
```

Comment:
Or would it be more consistent if I replace with 'discontinuous_no_defaults' instead?